setup/MsBuild.Engine.Corext/MsBuild.Engine.Corext.nuspec

```diff
@@ -16,6 +16,8 @@
     <file src="$X86BinPath$/Microsoft.Common.props" target="\Extensions\15.0" />
     <file src="$X86BinPath$/Microsoft.VisualStudioVersion.v15.Common.props" target="\Extensions\15.0" />
 
+    <file src="$X86BinPath$/NuGet.MSBuildSdkResolver.dll" target="v15.0/bin/SdkResolvers/NuGet.MSBuildSdkResolver" />
```

Comment:
what about x64?

Reply:
Resolvers only go in the 32 bit folder:
https://github.com/Microsoft/msbuild/blob/a955af860196952ba54a831cc2e57a91642d235a/src/Build/BackEnd/Components/SdkResolution/SdkResolverLoader.cs#L39-L40